### ISSUE TITLE
Immediate reload like on label for image

### ISSUE THREAD

**Issue by Aidavdw** [hyprwm/hyprlock#465]
### Description

Setting `text = cmd[update:(...)]` for label means the command is queried when hyprlock is first engaged.
Image objects work differently, where a (default) path is given with `path`, and the image can be reloaded with the command given by `reload_cmd`.
Setting dynamic images this way means that the default image is always loaded first, and then overridden. This means that the command must either poll, or a `SIGUSR2` signal must be sent by a background process immediately upon starting hyprlock (with some delay, requiring a fidgety `sleep (...)` to be able to set the image at the moment of starting hyperlock.

A system similar to the label setup is desirable for a image, where it can directly query a command/path, so that these extra steps don't have to be taken.

**Comment by c8dj2z26gj-a11y** [comment]
This would be amazing!

**Comment by PointerDilemma** [comment]
Yeah I agree, I wonder if we should just align the configuration format of image:path and background:path with the format for label:text.

**Comment by c8dj2z26gj-a11y** [comment]
That would be nice, I think it would make it more consistent to the coding of the other fields. And it feels like having them like a label would give people more freedom with the access to using scripts!
